packages/material-ui-lab/src/TreeItem/TreeItem.test.js

```diff
@@ -333,13 +333,10 @@ describe('<TreeItem />', () => {
 
       it('should work with programmatic focus', () => {
         const { getByTestId } = render(
-          <React.Fragment>
-            <div data-testid="start" tabIndex={0} />
```

Comment:
Unused element that was previously included due to c&p.

Reply:
I think I’ve removed this in the other PR. I’ve changed the tests quite a bit in the other one so kind of want to avoid merge conflicts 😂